ivadomed/loader/loader.py

```diff
@@ -69,21 +68,21 @@ def load_dataset(bids_df, data_list, transforms_params, model_params, target_suf
                                 soft_gt=soft_gt,
                                 is_input_dropout=is_input_dropout)
 
-    elif model_params[ModelParamsKW.NAME] == ConfigKW.HEMIS_UNET:
-        dataset = imed_adaptative.HDF5Dataset(bids_df=bids_df,
-                                              subject_file_lst=data_list,
-                                              model_params=model_params,
-                                              contrast_params=contrast_params,
-                                              target_suffix=target_suffix,
-                                              slice_axis=imed_utils.AXIS_DCT[slice_axis],
-                                              transform=tranform_lst,
-                                              metadata_choice=metadata_type,
-                                              slice_filter_fn=SliceFilter(**slice_filter_params,
-                                                                                            device=device,
-                                                                                            cuda_available=cuda_available),
-                                              roi_params=roi_params,
-                                              object_detection_params=object_detection_params,
-                                              soft_gt=soft_gt)
+    # elif model_params[ModelParamsKW.NAME] == ConfigKW.HEMIS_UNET:
+    #     dataset = imed_adaptative.HDF5Dataset(bids_df=bids_df,
+    #                                           subject_file_lst=data_list,
+    #                                           model_params=model_params,
+    #                                           contrast_params=contrast_params,
+    #                                           target_suffix=target_suffix,
+    #                                           slice_axis=imed_utils.AXIS_DCT[slice_axis],
+    #                                           transform=tranform_lst,
+    #                                           metadata_choice=metadata_type,
+    #                                           slice_filter_fn=SliceFilter(**slice_filter_params,
+    #                                                                                         device=device,
+    #                                                                                         cuda_available=cuda_available),
+    #                                           roi_params=roi_params,
+    #                                           object_detection_params=object_detection_params,
+    #                                           soft_gt=soft_gt)
```

Comment:
Hi @dyt811,
This PR is already merged but I have a question.
What is the reason for commenting these lines and not delete them?
(It cases conflicts on one of my PR, which I can resolve without issues but I was wondering)

Also, there are other places in the code and documentation where HeMis shows up (ex: in `models.py`, in the config file documentation, etc.), what is the plan for those?

Reply:
Feel free to delete them. I mainly wanted to keep this part so if we ever decide to resurrect the HeMIS related functions (or model after it), we have an idea what information goes where, parameter-wise. In practise it makes very little differences so feel free to delete if needed. I will just add a tag to note this timepoint in Git.

Reply:
Thanks @dyt811 for the info, I think I won't remove the commented lines because I prefer to keep my PR "in scope".

I don't have much experience in "deprecating" features, so I don't know how it is usually done.
It just seemed strange to me to have commented code without explanation as well as still referencing HeMis in the documentation.